components/card-list/css/_mixin.scss

```diff
@@ -0,0 +1,52 @@
+@mixin utrecht-card-list {
+  display: flex;
```

Comment:
Ik denk dat de responsiveness van deze component configureerbaar moet worden, of dat alleen de Card-component moet worden geëxporteerd , zodat we deze met andere Grid-componenten kunnen integreren.

[De huidige VTH website
](https://vth-programma-verslag.digitaal.utrecht.nl/nl) 

React Voorbeeld:

```tsx
import { Card } from '@utrecht/web-component-library-react';
import Image from 'next/image';

const CardData = [
  {
    title: 'Card 1',
    description: 'Card 1 description',
    image: {
      src: 'https://example.com/images/1',
      alt: 'Card 1 image',
      width: '100%',
      height: '100%',
    },
  },
  {
    title: 'Card 2',
    description: 'Card 2 description',
    image: {
      src: 'https://example.com/images/2',
      alt: 'Card 2 image',
      width: '100%',
      height: '100%',
    },
  },
  {
    title: 'Card 3',
    description: 'Card 3 description',
    image: {
      src: 'https://example.com/images/3',
      alt: 'Card 3 image',
      width: '100%',
      height: '100%',
    },
  },
];

export const ExampleCard = () => {
  return (
    <Grid>
      {CardData.map((card, index) => (
        <GridCell key={index} xs={12} sm={6} md={4} lg={3}>
          <Card
            title={card.title}
            description={card.description}
            image={
              <Image src={card.image.src} alt={card.image.alt} width={card.image.width} height={card.image.height} />
            }
          />
        </GridCell>
      ))}
    </Grid>
  );
};

```